src/importIssues.ts

```diff
@@ -136,6 +173,17 @@ export const importIssues = async (apiKey: string, importer: Importer) => {
     labelMapping[labelId] = labelResponse.issueLabelCreate.issueLabel.id;
   }
 
+  let userId = importAnswers.targetAssignee || 'me';
+  let assignee = ((await linear(`query {
```

Comment:
If it's a pick from a list, then there's no need to do this query as we already know the user ID.

Reply:
True. Will clean that up

Reply:
True. Will clean that up